code/modules/mining/abandoned_crates.dm

```diff
@@ -174,6 +174,11 @@
 /obj/structure/closet/crate/secure/loot/attack_animal(mob/user)
 	boom(user)
 
+/obj/structure/closet/crate/secure/loot/AltClick(mob/living/user)
+	if(user.incapacitated() || !Adjacent(user) || !ishuman(user))
```

Comment:
Nope -- it brings up the "enter code" screen but produces a "You don't have the dexterity" message if you enter anything.